Source/WebKit/NetworkProcess/cache/NetworkCacheKey.h

```diff
@@ -136,9 +137,7 @@ struct NetworkCacheKeyHash {
     static unsigned hash(const WebKit::NetworkCache::Key& key)
     {
         static_assert(SHA1::hashSize >= sizeof(unsigned), "Hash size must be greater than sizeof(unsigned)");
```

Comment:
FWIW I think this assert can go away now (since it is less precise than the assert in reinterpretCastSpanStartTo.

Reply:
That's a static assert though. Catching the issue at build time is always nice. I *think* reinterpretCastSpanStartTo would only assert at runtime?